tests/test_tod_ops.py

```diff
@@ -140,5 +140,28 @@ def test_basic(self):
                                        detrend='linear')
         self.assertEqual(sig1f.shape, tod['sig1d'].shape)
 
+class JumpfindTest(unittest.TestCase):
+    def test_jumpfinder(self):
+        """Test that jumpfinder finds jumps in white noise."""
+        tod = get_tod('white')
+        sig_jumps = tod.signal[0]
+        jump_locs = np.array([200, 400, 700])
+        sig_jumps[jump_locs[0]:] += 5
+        sig_jumps[jump_locs[1]:] -= 10
+        sig_jumps[jump_locs[2]:] -= 5
+
+        tod.wrap('sig_jumps', sig_jumps, [(0, 'samps')])
+
+        # Find jumps, using a conservative min_size here because the jumps
+        # are large and we want to avoid false positives
+        jumps = tod_ops.jumpfind(tod, signal=tod.sig_jumps, min_size=2)
```

Comment:
Calling this "conservative" is a but confusing.  I think it's not surprising that min_size should be a few times the white noise level (which is 1 here) to avoid false positives.

(This is another reason a min_sigma argument will be a little more powerful / less confusing.)